src/current/_includes/metrics-usage/tenant.consumption.write_bytes.md

```diff
@@ -0,0 +1,5 @@
+The number of RUs that were consumed due to KV writes, broken down by bytes. SQL statements are translated into lower-level KV write requests that are sent in batches. Batches may contain any number of requests. Requests can have a payload containing any number of bytes. Write operations are replicated to multiple storage processes (3 by default), with each replica counted as a separate write operation.Storage layer I/O is converted to Request Units using this equivalency:
```

Comment:
same as other comment re: "3 by default" and "learn more"

Reply:
kept `3 by default` for Serverless
updated `learn more`